charts/licensify/templates/ingress.yaml

```diff
@@ -1,26 +1,31 @@
-{{ $app := .Values.licensifyFrontend }}
-{{ $_ := set .Values "appName" $app.name }}
+{{- range .Values.apps }}
+{{- if .ingress.enabled }}
+{{ $_ := set $.Values "appName" .name }}
```

Comment:
Do you need to modify `.Values`? Might be cleaner to just define a variable, like `$appName := .name` (same as you're doing for `$svcPort`)?

Reply:
Ah so the reason is to make "appName" available for the "licensify.labels", by just adding it to the Values dict we pass it. (There might be a nicer way to do that.)

Reply:
Oh I see. Yeah, you probably wanna just change that partial template in _helpers.tpl to do what you want (probably just delete the `.Values` and pass it `.`) — but no biggie. As long as the labels are coming out the way you want them!